src/app/app.component.html

```diff
@@ -11,13 +11,24 @@
               <div *ngFor="let item of cart">
                 <div *ngIf="item.ingredient_name">
                   {{item.product_name}} med {{item.ingredient_name}} ({{item.price}} kr) x{{item.item_quantity}}
+                  <button class="btn-remove" id="remove_{{item.product_name}}-{{item.ingredient_name}}"
+                          (click)="removeProduct(item.product_id, item.product_name, item.price, item.ingredient_id, item.ingredient_name)">
+                    Ta bort
+                  </button>
                 </div>
                 <div *ngIf="item.ingredient_name == undefined">
                   {{item.product_name}} ({{item.price}} kr) x{{item.item_quantity}}
+                  <button class="btn-remove" id="remove{{item.product_name}}"
+                          (click)="removeProduct(item.product_id, item.product_name, item.price)">
```

Comment:
same here just send in the `item`?

Reply:
We will do this if/when we have time for some refactoring.  We started on it and it  starting breaking the tests and we didn't want to spend any more time on it, since have other stuff that needs to be done to get this up and working.